android/src/main/java/com/reactnative/adyendropin/AdyenDropInPayment.java

```diff
@@ -148,7 +151,12 @@ public void paymentMethods(String paymentMethodsJson) {
         this.cardConfiguration = cardConfiguration;
         Intent resultIntent = new Intent(this.getCurrentActivity(), this.getCurrentActivity().getClass());
         resultIntent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
-        this.dropInConfiguration = new DropInConfiguration.Builder(this.getCurrentActivity(), AdyenDropInPaymentService.class, "test_UV2LUSI7PZGGDEVR6ZEFJOAMVA5H5MYN").addCardConfiguration(cardConfiguration).build();
+        this.googlePayConfiguration = new GooglePayConfiguration.Builder(this.getCurrentActivity(), this.publicKey).setMerchantAccount("AlbertCZ").setGooglePayEnvironment(WalletConstants.ENVIRONMENT_TEST).setEnvironment(environment).build();
+        this.dropInConfiguration = new DropInConfiguration.Builder(this.getCurrentActivity(), AdyenDropInPaymentService.class, this.publicKey)
+                .addCardConfiguration(cardConfiguration)
+                .setEnvironment(Environment.TEST)
+                .addGooglePayConfiguration(this.googlePayConfiguration)
+                .build();
```

Comment:
Why you hard-coding env for payments? Is it always gonna be test? How we plan to dynamically support production one?